docs/excel/build-your-first-excel-add-in.md

```diff
@@ -10,7 +10,9 @@ You’ll start by creating a web app using HTML and JQuery. Then you'll create a
 
 ### Code it
 
-1- Create a folder on your local drive named QuarterlySalesReport (for example, C:\\QuarterlySalesReport). Save all files created in the steps that follow into this folder.
+#### Create the web app
+
+1-  Create a folder on your local drive named QuarterlySalesReport (for example, C:\\QuarterlySalesReport). Then within that folder, create a folder named QuarterlySalesReportWeb (for example, C:\\QuarterlySalesReport\\QuarterlySalesReportWeb). Save all of the web app files created in the steps that follow into this folder.
```

Comment:
Some minor edits - we can delete "Then" in the second sentence, and "of" in the third sentence. Also suggest changing "created" to "that you create".